src/__test__/pdf-router.test.js

```diff
@@ -0,0 +1,139 @@
+'use strict';
+
+import superagent from 'superagent';
+import { startServer, stopServer } from '../lib/server';
+import { createPdfMock, removePdfMock } from './lib/pdf-mock';
+
+const apiURL = `http://localhost:${process.env.PORT}`;
+
+describe('TESTING ROUTES AT /pdf', () => {
+  beforeAll(startServer);
+  afterEach(removePdfMock);
+  afterAll(stopServer);
+
+  describe('POST /pdf', () => {
+    test('should return 200 for successful POST', () => {
+      return createPdfMock()
+        .then((mockResponse) => {
+          const { token } = mockResponse.accountMock;
+          return superagent.post(`${apiURL}/pdf`)
+            .set('Authorization', `Bearer ${token}`)
+            .field('title', 'very important document')
+            .attach('pdf', `${__dirname}/assets/doc.pdf`)
+            .then((response) => {
+              expect(response.status).toEqual(200);
+              expect(response.body.title).toEqual('very important document');
+              expect(response.body._id).toBeTruthy();
+              expect(response.body.url).toBeTruthy();
+            });
+        })
+        .catch((error) => {
+          expect(error.status).toEqual(200);
+        });
+    });
+    test('should return 400 for bad POST request', () => {
+      return createPdfMock()
+        .then((mockResponse) => {
+          const { token } = mockResponse.accountMock;
+          return superagent.post(`${apiURL}/pdf`)
+            .set('Authorization', `Bearer ${token}`)
+            .attach('pdf', `${__dirname}/assets/doc.pdf`)
+            .then(Promise.reject)
+            .catch((error) => {
+              expect(error.status).toEqual(400);
+            });
+        });
+    });
+    test('POST should return 401 for bad token', () => {
```

Comment:
*This test actually fails because you include lines 52-53.*

In fact, if you're testing for just a 401 bad token anyway, think about if you really DO need to even create a mock to begin with. You could shorten this entire test block up a LOT by just doing a simple superagent post on its own with a bad token and handling it in a `.catch`.  Your test should pass once you clean this up. See me if you have questions.